data/json/items/comestibles.json

```diff
@@ -8453,6 +8453,62 @@
             "grow" : 85
         }
     },
+	{
+        "type" : "COMESTIBLE",
+        "id" : "cabbage",
+        "name" : "cabbage",
+        "weight" : 235,
+        "color" : "white",
+        "addiction_type" : "none",
+        "spoils_in" : 240,
+        "stim" : 0,
+        "container" : "null",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 2,
+        "healthy" : 0,
+        "addiction_potential" : 0,
+        "nutrition" : 15,
+        "description" : "A hearty head of crisp white cabbage.",
+        "price" : 100,
+        "material" : "veggy",
+        "tool" : "null",
+        "volume" : 3,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 4,
+        "bashing" : 0,
+        "fun" : 0
```

Comment:
Unless US has a different species of cabbage, cabbage eaten raw isn't exactly the most tasty thing to eat in bulk (too bitter). Doubly so if it's Chinese cabbage, which is even more bitter.

Currently carrots have -1 fun, so (raw) cabbage could easily have -1 or -2.

Could have minor health bonus, 1 or 2.

I don't remember if 4 charges at volume 3 means each charge has volume 4/3 or is it a 3 volume item of 12 charges. If the former (4/3 volume), it would be better to make both volume and charges 1.